docs/operate-and-deploy/changelog.md

```diff
@@ -6,6 +6,11 @@ description: Lists changes to the ksqlDB codebase
 keywords: ksqldb, changelog
 ---
 
+Version 0.29.0
+--------------
+
+- [ksqlDB v0.29.0 changelog](https://github.com/confluentinc/ksql/blob/master/CHANGELOG.md#0290-2023-04-24)
```

Comment:
The date included in this link needs to be updated, to be in sync with the other date above.